CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ dnf-automatic CHANGELOG
 This file is used to list changes made in each version of the
 dnf-automatic cookbook.
 
+2.0.5 (2023-11-27)
+------------------
+- Add AlmaLinux 9 Support
+
```

Comment:
Please remove this. Our automated system automatically updates this when we merge this.